### DISCUSSION TITLE
Week 11 hehe

### DISCUSSION THREAD

**Discussion by audreyhki** [USU-ITLS-x265-23-summer/class-repo#233] [General]
<h2>Progress</h2>
<h3>In week 10's update the project tasks I set out to accomplish were:</h3>

- [x] **Insert business logos:** I was able to accomplish this by adding an images folder to my project page and within my codespaces. Then I added the remaining business logos into the images folder and was able to insert the logos into my index.html code. 
- [x] **Insert additional images:** Adding the images folders, instead of storing images in Google Drive, made a big difference in streamlining and simplifying my operations and was obviously the correct way to link to my images. I appreciate @jakegros @coopsand and @kassyoveson for responding to my discussion call for help!
- [x] **Add business contact info:** I was able to retrieve and add the business's contact information and link to their Instagram at the bottom of the page.
- [ ] **Use CSS to style page further:** I somewhat successfully used CSS to further style the page but am still running into some roadblocks. I would not say I feel this task was fully accomplished and will be taking it on to week 12. I created new class selectors for my logos that didn't work and I'm not sure why.

<h3>For week 12 I'm aiming to:</h3>

- **Create working class selectors for my logos**
- **Continue to work on the CSS styling for the site**
- **Decide on whether or not the menu items should be capitalized or all lowercase** (the buisness's logo is all lowercase so the menu items look weird capitalized but also look wrong lowercased)
- **Figure out how to sync changes properly** (I posted about this problem on the discussion board but was still not able to remedy it. I am able to commit changes in code spaces but not able to sync them. This stresses me out and I am unsure if I am saving everything correctly. I plan on meeting with Professor Synder during office hours to see what I'm doing wrong)

**Comment by audreyhki** [comment]
@kensnyder

**Reply by kensnyder** [reply]
points assigned!

**Comment by derekscholes** [comment]
What are some of your CSS goals? What type of formatting are you using? I had some roadblocks as well then looked into flexbox and that helped me a lot with organization and format.

**Comment by CJ3315** [comment]
I would do capitalized as that would redirect the viewer's eyes to the menu items.

**Reply by audreyhki** [reply]
Thanks for your input, I decided to go with the capitalized version because of your suggestion!

**Comment by kassyoveson** [comment]
Audrey. I had similar problems with getting my changes to sync. I used the visual studio code to view my live updates. I had to have a neighbor come and help me get connected to the GitHub repository. He also taught me how to get status and push the data to the repository. I never would have been able to do it without his help.
